components/bl-radio-buttons-list-component/src/input.js

```diff
@@ -0,0 +1,21 @@
+import { CheckedIcon } from './checked-icon';
+import { UncheckedIcon } from './unchecked-icon';
+
+export function Input({ value, checked, handleChange }) {
```

Comment:
Input, RadioButton and Icons components could be moved into one file `components.js`